numpy/core/src/umath/loops_trigonometric.dispatch.c.src

```diff
@@ -0,0 +1,230 @@
+/*@targets
+ ** $maxopt $werror baseline
```

Comment:
```suggestion
 ** $maxopt baseline
```
remove treating warnings as errors after the CI pass the tests

Reply:
CI is passing

Reply:
Done, I temporarily use this policy during the development to detect any warnings.